frontend/public/components/resource-dropdown.tsx

```diff
@@ -137,11 +143,17 @@ const ResourceListDropdown_: React.SFC<ResourceListDropdownProps> = (props) => {
       menuClassName="dropdown-menu--text-wrap"
       className={classNames('co-type-selector', className)}
       items={allItems}
-      title={allItems[selectedKey]}
+      title={
+        <div key={'title-resource'}>
```

Comment:
Is `key` needed? If it is, it should be

```suggestion
        <div key="title-resource">
```

But would be better to leave off if not required.

Reply:
Key is needed